roles/docker/tasks/package_docker.yml

```diff
@@ -97,7 +97,8 @@
       {% if docker_log_driver is defined  %} --log-driver {{ docker_log_driver }}{% endif %}\
       {% if docker_log_options is defined %} {{ docker_log_options |  oo_split() | oo_prepend_strings_in_list('--log-opt ') | join(' ')}}{% endif %}\
       {% if docker_options is defined %} {{ docker_options }}{% endif %}\
-      {% if docker_disable_push_dockerhub is defined %} --confirm-def-push={{ docker_disable_push_dockerhub | bool }}{% endif %}'"
+      {% if docker_disable_push_dockerhub is defined %} --confirm-def-push={{ docker_disable_push_dockerhub | bool }}{% endif %}\
+      {% if openshift.node is defined and openshift.node.sdn_mtu is defined %} --mtu={{ openshift.node.sdn_mtu }}{% endif %}'"
```

Comment:
This adds the option to OPTIONS in /etc/sysconfig/docker, but in theory network-specific options are supposed to go into DOCKER_NETWORK_OPTIONS in /etc/sysconfig/docker-network. (I don't know why really, but, eg, openshift-sdn originally didn't do it that way, and someone filed a bug about it, https://github.com/openshift/openshift-sdn/issues/52.)